includes/admin/helpers/csv/class-csv-import.php

```diff
@@ -653,7 +653,7 @@ private function import_row( $data ) {
 			}
 		}
 
-		if ( $state->images ) {
+		if ( is_array( $state->images ) && ! empty( $state->images ) && ! empty( $state->images[0] ) ) {
```

Comment:
Thank you @shervElmi ! Does this pass without "is_array( $state->images )" too?

Reply:
You're welcome, I have made the necessary changes in my recent commit, and it now passes without using "is_array( $state->images )".

Reply:
Thanks! That's nice to have it a bit shorter now